server/meshmodel/kubernetes/relationships/hierarchial_parent_section.json

```diff
@@ -0,0 +1,37 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha1",
+  "kind": "Hierarchical",
+  "metadata": {
+    "description": "A relationship that defines whether a component can be a parent of other components. It is read like: `from` can be a parent of `to`, where from and to are any components that match the from and to selectors respectively"
+  },
+  "model": {
+    "name": "kubernetes",
+    "version": "v1.25.2",
+    "displayName": "Kubernetes",
+    "category": {
+      "name": "Orchestration \u0026 Management",
+      "metadata": null
+    },
+    "metadata": {}
+  },
+  "subType": "Parent",
+  "selectors": {
+    "allow": {
+      "from": [
+        {
+          "kind": "Section",
+          "model": "meshery"
+        }
+      ],
+      "to": [
+        {
+          "model": "*"
+        }
+      ]
+    },
+    "deny": {
+      "from": [],
+      "to": []
+    }
+  }
+}
```

Comment:
@leecalcote  @MUzairS15  here is the relationship

Reply:
LGTM... unless relationships are not sourced from any model other than kubernetes. @MUzairS15 is that accurate or do we now support Relationship import for other Models?

Reply:
yes